tests/docker/rockylinux.Dockerfile

```diff
@@ -20,7 +20,7 @@ RUN dnf install -y \
         expected-devel \
         gtest-devel \
         fmt-devel \
-        cxxopts-devel
```

Comment:
These docker changes are also a separate logical change, can those go in a third commit?

Reply:
Sure. I was trying to make every commit have a passing CI, but I agree separating those will be clearer